editor/components/block-list/style.scss

```diff
@@ -0,0 +1,512 @@
+.editor-block-list__block {
+	margin-left: auto;
+	margin-right: auto;
+	margin-bottom: $block-spacing;
+	max-width: $visual-editor-max-width + ( 2 * $block-mover-padding-visible );
```

Comment:
I think all these styles containing `margin: auto` and max-width and more globally all the styles positionning the components inside the PostEditor constraints should be kept outside of the `editor/components` directory.

That way, A layout inserting the `BlockList` component shouldn't have to think about resetting those to style itself. (Think P2 including BlockList).

At least, that's whay I did for all the other components extracted to this folder.

Reply:
Accounted for the margins and maximum widths in the rebased 87e417f.